convert.c

```diff
@@ -969,6 +1081,30 @@ static int ident_to_worktree(const char *path, const char *src, size_t len,
 	return 1;
 }
 
+static struct encoding *git_path_check_encoding(struct attr_check_item *check)
+{
+	const char *value = check->value;
+	struct encoding *enc;
+
+	if (ATTR_UNSET(value))
+		return NULL;
+
+	for (enc = encoding; enc; enc = enc->next)
```

Comment:
If the code isn’t C98 and that’s not mandated by style otherwise, `struct encoding *enc = encoding` could be put directly into the for loop’s init statement.

Reply:
Git wants its variables defined first.

Reply:
Okay!